### ISSUE TITLE
Metapackages: Prevent installation into non-netstandard20 projects

### ISSUE THREAD

**Issue by thaxy** [aspnet/EntityFrameworkCore#9654]
When I try to add the NuGet package ``Microsoft.EntityFrameworkCore.Sqlite`` to a newly created Console Application with .NET Framework 4.5.2 it has no effect. After I accept the license I cannot see any added references. There is no pop up window which asks to add additional dependencies.

The ``packages`` directory of my project repository has one single folder called ``Microsoft.EntityFrameworkCore.Sqlite.2.0.0`` which contains only one file named ``Microsoft.EntityFrameworkCore.Sqlite.2.0.0.nupkg``

I am using Visual Studio 2017 Enterprise 15.3.3. Same behaviour under Visual Studio 2015 Professional Update 3.

Update:
  * It works when I upgrade my Project to .NET Framework 4.6.x
_________________________________________________
Attempting to gather dependency information for package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0' with respect to project 'ConsoleApp3', targeting '.NETFramework,Version=v4.5.2'
Gathering dependency information took 138.86 ms
Attempting to resolve dependencies for package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0'
Resolved actions to install package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0'
Retrieving package 'Microsoft.EntityFrameworkCore.Sqlite 2.0.0' from 'nuget.org'.
Adding package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0', which only has dependencies, to project 'ConsoleApp3'.
Adding package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0' to folder 'C:\Users\x x\Documents\Visual Studio 2017\Projects\ConsoleApp3\packages'
Added package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0' to folder 'C:\Users\x x\Documents\Visual Studio 2017\Projects\ConsoleApp3\packages'
Added package 'Microsoft.EntityFrameworkCore.Sqlite.2.0.0' to 'packages.config'
Successfully installed 'Microsoft.EntityFrameworkCore.Sqlite 2.0.0' to ConsoleApp3
Executing nuget actions took 89.23 ms
Time Elapsed: 00:00:00.2598284
========== Finished ==========

**Comment by bricelam** [comment]
Hmm.. We might need to add a stub `lib\netstandard2.0\_._` file to get NuGet to error.

**Comment by bricelam** [comment]
(Probably needed in `Microsoft.Data.Sqlite` too.)

**Comment by ajcvickers** [comment]
@thaxy EF Core only supports .NET Framework 4.6.1 and up, since this is the lowest framework version that implements .NET Standard 2.0.  As @bricelam commented, we'll look into why NuGet doesn't say something useful.

**Comment by thaxy** [comment]
Thanks for the help. There was one more weird behavior I noticed. When I successfully installed EF Core via NuGet to my application (.NET Framework 4.6.1) everything worked like expected and under the NuGet browser I could see the newly installed packages. But when I switched the framework version to e.g. 4.5.2 the NuGet list of installed packages got messed up - I saw about 20 dependencies. When I switched back to the original framework version this state persisted.
Does this make sense to you guys? I could upload a picture to make this more clear in 12h.
